deploy/internal/loadPackages.js

```diff
@@ -1,19 +0,0 @@
-const fs = require("fs");
```

Comment:
deploy/publish.js use deploy/internal/loadPackages.json
so loadPackages may not be removed

Reply:
ok I revert it